pkg/operator/staticpod/controller/monitoring/manifests/service-monitor.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: {{ .ServiceMonitorName }}
```

Comment:
question: do we want the name match the namespace? if yes, I can drop this parameter, but it felt weird.

Reply:
(also I dunno if the name of the service monitor plays any role as this is namespaces resource, we can just call this "monitor" ;-)